docs/getting-started/setup/using-lts-edition.md

```diff
@@ -0,0 +1,335 @@
+
+## What is LTS Edition?
+
+CKEditor&nbsp;5 LTS Edition is designed for teams that need **long-term stability without sacrificing security**. Released once every two years, each LTS version provides up to 3 years of guaranteed security and compatibility fixes with **zero breaking changes**.
+
+### Key advantages over regular releases
+
+<table>
+	<thead>
+		<tr>
+			<th>LTS Edition</th>
+			<th>Regular releases</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td><strong>Once every 2 years</strong></td>
+			<td>Major version every 2–3 months</td>
+		</tr>
+		<tr>
+			<td><strong>3-year security warranty</strong></td>
+			<td>Security fixes only in the latest version</td>
+		</tr>
+		<tr>
+			<td><strong>Third-party API changes assurance</strong></td>
+			<td>No third-party compatibility fixes</td>
+		</tr>
+		<tr>
+			<td><strong>Zero breaking changes</strong></td>
+			<td>Minor breaking changes allowed</td>
+		</tr>
+		<tr>
+			<td><strong>No forced upgrades for 3 years</strong></td>
+			<td>Requires regular major updates</td>
+		</tr>
+	</tbody>
+</table>
+
+### Perfect for enterprise environments
+
+LTS Edition lets you **stick to a single major version for up to 3 years** without compromising on security or quality. You get:
+
+* **6 months of active development** &ndash; New features, enhancements, and bug fixes
+* **2.5 years of maintenance phase** &ndash; Security and critical/high third-party compatibility updates (for example, with browsers or Microsoft Word/Excel)
+* **Predictable update cycles** &ndash; Plan deployments without surprise breaking changes
+* **Focus on your business** &ndash; Less time managing editor updates, more time building features
+
+This means enterprise teams in regulated industries, government, education, or any environment where stability is crucial can deploy once and maintain security without the overhead of frequent major version migrations.
+
+## LTS Edition release schedule
+
+{@img assets/img/lts-schedule.png 800 Timeline chart of CKEditor&nbsp;5 release schedule from Q3 2025 to Q3 2031, showing major versions v47 to v58. LTS releases (v47, v51, v55) have a 6-month Active phase in green followed by a 2.5-year Maintenance phase in yellow. Regular releases (v48, v49, v50, v52, v53, v54, v56, v57, v58) each have shorter Active phases of about 6 months in green, with no Maintenance phase. LTS releases overlap by one year, allowing migration from one LTS line to the next.}
+
+<!-- Source: https://miro.com/app/board/uXjVJCW7oaw=/ -->
+
+Legend:
+
+* **Active phase**: New features and bug fixes. (~6 months)
+	* In LTS releases: no breaking changes allowed.
+	* In Regular releases: minor breaking changes allowed.
+* **Maintenance phase**: Security and critical/high third‑party compatibility updates. Available to LTS Edition customers only. (~2.5 years)
+
+### Schedule details
+
+Schedule details of the **LTS releases**:
+
+- **Cadence**: One LTS release every 2 years (early October).
```

Comment:
Could we unify the list mark symbol? https://github.com/ckeditor/ckeditor5/pull/19171/files#diff-b8daf6527538a321b53d890b46f7ad17b1c5d695f56acebdd1159df266509b25R56-R59 use `*`.